### DISCUSSION TITLE
3.1.0 version

### DISCUSSION THREAD

**Discussion by tonylu110** [tonylu110/uyou-todo-electron#51] [Announcements]
- From this version onwards the default mode is `Note Mode`
- Added local backup ToDo items and categories
- Added delete all completed ToDo items
- Fixed Sidebar don't Shrink
- Fixed font style use (only windows)

<hr /><em>This discussion was created from the release <a href='https://github.com/tonylu110/uyou-todo-electron/releases/tag/3.1.0'>3.1.0 version</a>.</em>

**Comment by lingyuajnzi** [comment]
我提个建议，我想增加一个功能，就是可以嵌入桌面。另外，分离设置窗口。还有切换模式不要重启生效，应该拉大窗口就可以生效。平时做小窗嵌入桌面（参考小黄条），ctrl+D,还会保留在桌面。。如果需要大量编辑时，可以到标准模式。界面很美观。但停留在桌面时，我想激活的毛玻璃很漂亮，不要变灰了。是什么界面就怎样吧，不要变灰了。或者把这个变灰的功能留给用户选择。非常喜欢你这个软件。

**Comment by lingyuajnzi** [comment]
置顶功能和嵌入功能是矛盾的，希望给用户选择。

**Comment by lingyuajnzi** [comment]
![image](https://github.com/user-attachments/assets/270d6c10-f48a-460c-b52a-3ca745b8e704)
点击无法删除分类

**Comment by lingyuajnzi** [comment]
![image](https://github.com/user-attachments/assets/52ce7c32-ba8a-4442-b9f4-151bc2d9164c)
